cartoframes/data/clients/data_obs_client.py

```diff
@@ -9,10 +9,10 @@
 
 from carto.exceptions import CartoException
 
-from .. import utils
-from .. import context
-from ..auth import get_default_credentials
-from ..data.dataset import Dataset
+from cartoframes.utils import utils
+from cartoframes.lib import context
+from cartoframes.auth import get_default_credentials
+from cartoframes.data.dataset.dataset import Dataset
```

Comment:
I see in other places `from cartoframes.data import Dataset` 

I assume `cartoframes.data.dataset.dataset` is the actual absolute path, and `cartoframes.data` is the module we are exporting right?

Reply:
> I see we are using absolute paths instead of relative ones. What's the reason for that?

My new IDE is using absolute ones. I don't have a preference between both.

> I assume cartoframes.data.dataset.dataset is the actual absolute path, and cartoframes.data is the module we are exporting right?

Right. We are exposing Dataset class in the `__init__` file in `cartoframes.data`

Reply:
I guess we had a conversation about this and we decided to go with relative paths (looked like the more pythonic way). I was wondering if the change was related to some circular dependency issue or so but if not would it be a pain to revert back to relative ones?

Reply:
Let's try! I think with `circular dependency` pain is not bigger with relative ones

Reply:
🤞

Reply:
done